float64[] descriptor 
---
LamaDescriptorIdentifier id

